prime-dotnet-webapi/ApiDbContext.cs

```diff
@@ -162,35 +165,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             }
             #endregion
 
-            modelBuilder.ApplyConfiguration(new EnrolleeConfiguration());
-            modelBuilder.ApplyConfiguration(new CollegeConfiguration());
-            modelBuilder.ApplyConfiguration(new LicenseConfiguration());
-            modelBuilder.ApplyConfiguration(new CollegeLicenseConfiguration());
-            modelBuilder.ApplyConfiguration(new PracticeConfiguration());
-            modelBuilder.ApplyConfiguration(new CollegePracticeConfiguration());
-            modelBuilder.ApplyConfiguration(new JobNameConfiguration());
-            modelBuilder.ApplyConfiguration(new OrganizationTypeConfiguration());
-            modelBuilder.ApplyConfiguration(new StatusConfiguration());
-            modelBuilder.ApplyConfiguration(new StatusReasonConfiguration());
-            modelBuilder.ApplyConfiguration(new CountryConfiguration());
-            modelBuilder.ApplyConfiguration(new ProvinceConfiguration());
-            modelBuilder.ApplyConfiguration(new SelfDeclarationTypeConfiguration());
-
-            modelBuilder.ApplyConfiguration(new PrivilegeTypeConfiguration());
-            modelBuilder.ApplyConfiguration(new PrivilegeGroupConfiguration());
-            modelBuilder.ApplyConfiguration(new PrivilegeConfiguration());
-            modelBuilder.ApplyConfiguration(new DefaultPrivilegeConfiguration());
-            modelBuilder.ApplyConfiguration(new AssignedPrivilegeConfiguration());
-            modelBuilder.ApplyConfiguration(new EnrolleeProfileVersionConfiguration());
-
-            modelBuilder.ApplyConfiguration(new GlobalClauseConfiguration());
-            modelBuilder.ApplyConfiguration(new UserClauseConfiguration());
-            modelBuilder.ApplyConfiguration(new LicenseClassClauseConfiguration());
-            modelBuilder.ApplyConfiguration(new LicenseClassClauseMappingConfiguration());
-
-            modelBuilder.ApplyConfiguration(new BusinessEventTypeConfiguration());
-
-            modelBuilder.ApplyConfiguration(new VendorConfiguration());
+            modelBuilder.ApplyConfigurationsFromAssembly(typeof(ApiDbContext).Assembly);
```

Comment:
Rather than loading every file explicitly, we can use ApplyConfigurationsFromAssembly() to load all configuration files inheriting from IEntityTypeConfiguration.